[nnc] Loweings for flatten, xnnpack prepack op

IvanKobzarev opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* __->__ #68470

Differential Revision: [D32545261](https://our.internmc.facebook.com/intern/diff/D32545261)

IvanKobzarev committed:
[nnc] Loweings for flatten, xnnpack prepack op

[ghstack-poisoned]

ZolotukhinM reviewed (approved):
Looks good, but at some point we would need to add tests too! :)

IvanKobzarev committed:
Update on "[nnc] Loweings for flatten, xnnpack prepack op"




[ghstack-poisoned]

IvanKobzarev commented:
@ivankobzarev has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D32545261).

IvanKobzarev committed:
Update on "[nnc] Loweings for flatten, xnnpack prepack op"


Differential Revision: [D32545261](https://our.internmc.facebook.com/intern/diff/D32545261)

[ghstack-poisoned]

IvanKobzarev commented:
@ivankobzarev has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D32545261).
